addons/featured-dangos-v2/addon.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Featured dangos",
+  "description": "A Scratch Addons April Fools joke from 2023 that added a row to the front page featuring dango-related projects.",
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "style.css",
+      "matches": ["https://scratch.mit.edu/"]
+    }
+  ],
+  "tags": ["community", "easterEgg"],
+  "versionAdded": "1.33.0"
```

Comment:
Apparently easter egg addons might appear under "other new addons" which is probably unintended.

Reply:
It's not wrong though, I simply didn't expect this 🤔 

![image](https://github.com/ScratchAddons/ScratchAddons/assets/17484114/6e7968b6-d68d-4892-8f70-7439e002cb7f)

Reply:
Should the version added be reverted to prevent this?

Reply:
Yeah, I guess that's easier than changing the settings page code.